### DISCUSSION TITLE
navmesh

### DISCUSSION THREAD

**Discussion by unicomp21** [RaananW/babylonjs-webpack-es6#26] [Q&A] [answered]
@RaananW @DiogoNeves is this the proper way to use navmesh in babylonjs?

https://github.com/firdodev/tetrisBabylon/blob/68dca50cf83dd76a4873a0e1c88de97d20455173/package.json

**Comment by unicomp21** [comment, marked as answer]
nevermind, figured it out.  the docs need to highlight the need to call "await" in order to load the library before using it.

**Reply by DiogoNeves** [reply]
Hi @unicomp21 sorry for the delay. Thanks for the mention! I'm new to the project too, I've only done a very small contribution.  
You probably came from the forums already, but just in, you should be able to find help at http://forum.babylonjs.com/ 

Have you seen this doc page too? https://doc.babylonjs.com/features/featuresDeepDive/crowdNavigation/createNavMesh

**Reply by DiogoNeves** [reply]
Regarding the `await`, can you point me to the code where you use it?

**Comment by unicomp21** [comment]
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
import Recast from "recast-detour"

followed by:
await Recast();

before using the navmesh stuff.
